### PR TITLE
Replace flash-based copy to clipboard script

### LINKED ISSUE #26: Replace zeroclipboard with clipboard (doesn't require flash)
https://www.npmjs.com/package/clipboard

### PR DESCRIPTION
**Pull request by maggiewachs**
Closes #26

### CODE REVIEW

**Comment by zachleat** [comment]
Unrelated to this PR I did open https://github.com/filamentgroup/X-rayHTML/issues/28 just now because I noticed we’re doing manual copies of dependencies into `libs/`

@maggiewachs I wasn’t able to get this to work in Firefox? The Copy button wasn’t showing and I had this JS error:
image

**Comment by zachleat** [comment]
It might be worth upgrading shoestring here? Looks outdated

File: index.html [hunk 1]
```diff
@@ -11,102 +11,157 @@
...
+		if( prism && "Prism" in window ) {
+			$( ".prism" ).find( "code" ).addClass( "language-markup" );
+			Prism.highlightAll();
+		}
+
+		// Clipboard
+		var copy = !!~e.target.getAttribute( "class" ).indexOf( "xray-copy" );
+
+		if( copy && "Clipboard" in window ) {
+			var $target = $( event.target );
```
Comment (zachleat) [inline index.html]: Oh! I think this should just be `e` instead of `event`

Comment (maggiewachs) [inline index.html]: I can fix -- and move the lib js into the package json.

**Comment by maggiewachs** [comment]
@zachleat fixed & ready for review. Thanks!

**Comment by zachleat** [comment]
Tested locally, looks good!

**Pull request merged**